yt_dlp/extractor/common.py

```diff
@@ -2013,6 +2022,18 @@ def extract_media(x_media_line):
             if not (media_type and group_id and name):
                 return
             groups.setdefault(group_id, []).append(media)
+            # <https://tools.ietf.org/html/rfc8216#section-4.3.4.1>
+            if media_type == 'SUBTITLES':
+                lang = media['LANGUAGE']  # XXX: normalise?
+                url = format_url(media['URI'])
+                sub_info = {
+                    'url': url,
+                    'ext': determine_ext(url),
+                }
+                if sub_info['ext'] == 'm3u8':  # XXX
```

Comment:
why xxx?

Reply:
Back when I wrote this, I just blindly assumed all HLS subtitles are WebVTT instead of consulting the spec (which says it indeed is the case). I guess it can be removed.